### DISCUSSION TITLE
Multiple imputation example not working

### DISCUSSION THREAD

**Discussion by mhoenicka** [vincentarelbundock/marginaleffects#1100] [General]
Hi,
I've been trying to adapt the Amelia sample code on the multiple imputations vignette (https://marginaleffects.com/vignettes/multiple_imputation.html) to my own project. As this resulted in an error message, I copypasted the provided example in the Amelia section and ran that one. Same error message:

> mod_amelia <- with(dat_amelia, lm(Petal.Width ~ Sepal.Length * Sepal.Width + Species))
Error in eval(predvars, data, env) : object 'Petal.Width' not found

My environment at work is provided below. The same error occurs on an up-to-date FreeBSD system at home, I can supply the sessionInfo() output of my homebox too if needed.
Any idea how to fix this?
Markus

> sessionInfo()
R version 3.6.3 (2020-02-29)
Platform: x86_64-pc-cygwin (64-bit)
Running under: CYGWIN_NT-10.0 PC6187 3.1.6(0.340/5/3) 2020-07-09 08:20 x86_64 Cygwin

Matrix products: default
LAPACK: /usr/lib/R/modules/lapack.dll

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] Amelia_1.7.6           Rcpp_1.0.4.6           marginaleffects_0.19.0

loaded via a namespace (and not attached):
[1] compiler_3.6.3    cli_3.6.1         generics_0.1.3    foreign_0.8-75
[5] data.table_1.12.8 rlang_1.1.0

**Comment by vincentarelbundock** [comment]
There is no `marginaleffects` code at all in the example you give. This appears to be unrelated to `marginaleffects`, so I'm not sure what can be discussed here, exactly. Maybe this question should be targeted to an `Amelia` forum?

FWIW, I copied the code from the vignette in a fresh R session and everything works as intended.

**Reply by mhoenicka** [reply]
Well, the next line would have been a call to avg_slopes() if the Amelia code hadn't failed on me - I failed to mention that I want to use avg_slopes() on imputed datasets. But you are right, of course. I tried the sample code in a fresh environment on my FreeBSD box, and sure enough it worked like a charm. I'll trace back my steps and see why it failed in my work environment.
Thx, Markus

**Reply by vincentarelbundock** [reply]
Yep yep, makes sense. Glad it now seems to be working on at least one machine!
